packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/SVGImage/SVGImage.spec.js

```diff
@@ -0,0 +1,46 @@
+import { expect } from 'chai'
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+
+import SVGImage from './SVGImage'
+
+describe('SVGImage', function () {
+  it('should render an image', function () {
+    render(
+      <SVGImage
+        naturalHeight={100}
+        naturalWidth={100}
+        src='https://some.domain/image.jpg'
+        subjectID='1234'
+      />
+    )
+
+    const image = screen.getByRole('img')
```

Comment:
```suggestion
    const image = screen.getByRole('img', { name: 'Subject 1234' })
```

I've been trying to get into the habit of querying by role and name, to check that an element is appropriately named in the accessibility tree.